float32 joint0
float32 joint1
float32 joint2
float32 joint3
float32 joint4
---
int8 status
